releasenotes/notes/add-timing-output-to-dwave-sample-e3adb8b197921e8d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add wall-clock and QPU timing details to ``dwave sample`` output.
```

Comment:
If your hyphenated "wall-clock" is an adjective on details then so should be "QPU-timing". Perhaps "Add wall time and QPU timing to ``dwave sample`` output."